dev/com.ibm.ws.jaxrs.2.0_fat/test-applications/paramconverter/src/com/ibm/ws/jaxrs/fat/paramconverter/ClientTestServlet.java

```diff
@@ -244,7 +244,7 @@ public void testNoPublicConstructorListObject() throws Exception {
         assertEquals("expected: " + expected + " actual: " + actual, expected, actual);
     }
 
- // Waiting for feature 15234    @Test
+ // Waiting for feature 15234 (CXF change:  https://github.com/apache/cxf/pull/809 did not resolve)   @Test
```

Comment:
I think we just delete this test method instead of leaving it commented out.

Reply:
Agreed.  I was waiting to ensure the behavior was the same on Jersey (and it is).